feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -219,8 +219,9 @@ func TestHardwarePort(t *testing.T) {
 	t.Logf("For interface %s, HardwarePort is %s", dp.Name(), val)
 
 	// Verify HardwarePort is a component of type PORT.
-	typeGot := gnmi.Get(t, dut, gnmi.OC().Component(val).Type().State())
-	if typeGot != portType {
+	typeGotV := gnmi.Lookup(t, dut, gnmi.OC().Component(val).Type().State())
+	typeGot, present := typeGotV.Val()
+	if present && typeGot != portType {
```

Comment:
This change was not included in the PR description / not part of the deviation, it appears

Reply:
Thanks. Reverted.